Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CorrelationAttributeNormalizer.java

```diff
@@ -164,6 +165,9 @@ private static String normalizeEmail(String data) throws CorrelationAttributeNor
     private static String normalizePhone(String data) throws CorrelationAttributeNormalizationException {
         if (data.matches("\\+?[0-9()\\-\\s]+")) {
             String phoneNumber = data.replaceAll("[^0-9\\+]", "");
+            if (phoneNumber.length() <= MIN_PHONE_LENGTH) {
+                throw new CorrelationAttributeNormalizationException(String.format("Data was expected to be a valid phone number: %s", data));
+            }
```

Comment:
With the phone number validation move, I'm now getting stack traces in the log every time it finds a short or otherwise invalid phone number. I believe we were previously just not recording them - they can be valid but are not interesting for correlation.

Reply:
@rcordovano Should this logic be moved back over? I don't see why this shouldn't be here (to enforce this check across all modules). My first reaction is that we probably shouldn't log these things so heavily, who cares if its not valid?

Reply:
@rcordovano Do we study production logs heavily, that's the only way I can see this being useful (i.e we find email formats that are actually real and common in production)?